README.md

```diff
@@ -26,22 +35,39 @@ package main
 import (
 	"github.com/aws/aws-lambda-go/events"
 	"github.com/aws/aws-lambda-go/lambda"
-	wflambda "github.com/wavefronthq/wavefront-lambda-go" // Import this library
+	wflambda "github.com/retgits/wavefront-lambda-go" // Import this library
 )
 
+var wfAgent = wflambda.NewWavefrontAgent(&wflambda.WavefrontConfig{})
+
 func handler() (string, error){
 	return "Hello World", nil
 }
 
 func main() {
-	// Wrap the handler with wflambda.Wrapper()
-	lambda.Start(wflambda.Wrapper(handler))
+	// Wrap the handler with wfAgent.WrapHandler()
+	lambda.Start(wfAgent.WrapHandler(handler))
 }
 ```
 
-## Standard Point Tags
+## Configuration
+
+The `wfAgent` variable in the previous sample can be configured using both environment variables, as well as values passed into it using the `WavefrontConfig` struct. If both WavefrontConfig and environment variables have a value for a specific setting, the environment variable takes precedence. The configuration options you can set are:
+
+* **Enabled** (`*bool`): Enabled indicates whether metrics are sent to Wavefront. The environment variable `WAVEFRONT_ENABLED` is also used for this setting.
```

Comment:
We also need a "REPORT_STANDARD_METRICS" flag, meaning standard lambda metrics will not be sent to wavefront if this flag is set to false. (This was supported in earlier library using REPORT_STANDARD_METRICS env variable).

Reply:
I'd be inclined to not have that flag. These AWS Lambda metrics are essentially the bare minimum you'd need to manage your functions so not reporting them doesn't make a lot of sense to me. My suggestion would be to keep it as is for now, and we can open a discussion issue on GitHub to see what thoughts are from the community.

Reply:
I agree with you. Our primary goal for having a lambda wrapper is to :
1. Provide a way for easily monitoring their lambda's
2. Also provide a way for user to easily send custom metrics from their Lambda's to wavefront.

This flag will enable users for case #2. Users can also use this flag to test their lambda's in dev environment etc. without having the additional cost for ingesting standard lambda metrics.

Given this was a part of initial MVP , If its not too much effort, I would suggest to not break backward compatibility with this.